ports/raspberrypi/boards/raspberry_pi_pico2/mpconfigboard.mk

```diff
@@ -10,3 +10,4 @@ CHIP_FAMILY = rp2
 EXTERNAL_FLASH_DEVICES = "W25Q32JVxQ"
 
 CIRCUITPY__EVE = 1
+CIRCUITPY_SSL = 1
```

Comment:
Same for this?

Reply:
this should probably be taken out, or at least discussed separately.